persistqueue/tests/test_sqlqueue.py

```diff
@@ -454,6 +455,18 @@ def test_unique_dictionary_serialization_msgpack(self):
         queue.put({"bar": 2, "foo": 1})
         self.assertEqual(queue.total, 1)
 
+    def test_unique_dictionary_serialization_cbor2(self):
+        queue = UniqueQ(
+            path=self.path,
+            multithreading=True,
+            auto_commit=self.auto_commit,
+            serializer=serializers_cbor2
+        )
+        queue.put({"foo": 1, "bar": 2})
+        self.assertEqual(queue.total, 1)
+        queue.put({"bar": 2, "foo": 1})
+        self.assertEqual(queue.total, 1)
```

Comment:
Better to verify the content in the queue as well

Reply:
None of the other tests (pickle, msgpack, json) verify the content in the queue.  I presume this is because this particular test is testing that duplicate items are only queued once for `UniqueQ`.  I also presume that verifiying the content is handled in `test_queue.py`

Reply:
You're right, we have more test to cover `serialization` scenario for file based queues, maybe we can enhance the coverage next time for sqlite base queues